index.js

```diff
@@ -18,3 +22,62 @@ closedMenu.addEventListener('click', close);
 for (let i = 0; i < menuLists.length; i += 1) {
   menuLists[i].addEventListener('click', close);
 }
+
+const projects = {
+  name: 'multi-post stories',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+  img: 'images/popup.png',
+  language1: 'css',
+  language2: 'html',
+  language3: 'bootstrap',
+  language4: 'Ruby',
```

Comment:
- [ ] As per the project requirements(see image below), and best practices, please make sure each object in the projects array contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Please kindly add link to live version and link the the source code.

![Meri](https://user-images.githubusercontent.com/65852621/136193891-559001b3-4fed-43cf-849b-a908a5db5bd6.png).